### DISCUSSION TITLE
Future directions regarding optimization

### DISCUSSION THREAD

**Discussion by x02Sylvie** [ggml-org/llama.cpp#718] [Ideas]
I'm kinda curious how much more we can squeeze out of llama on cpu, what other directions can we take.

Just casual topic for brainstorming

**Comment by MillionthOdin16** [comment]
They're always coming up with new approaches :) it's cool to see how some small changes can have a huge impact.

**Comment by PriNova** [comment]
What I really like to operating on is the evolved RNN from this https://github.com/BlinkDL/ChatRWKV 
It looks very promising in regards to "unlimited" context window.
